barman/cli.py

```diff
@@ -67,8 +186,11 @@ def check_target_action(value):
     raise ArgumentTypeError("'%s' is not a valid recovery target action" % value)
 
 
-@named("list-servers")
-@arg("--minimal", help="machine readable output")
+@command([
+        argument("--minimal",
+                 help="machine readable output")
+        ],
+        cmd_aliases=["list-server"])
 def list_servers(minimal=False):
```

Comment:
This should accept `args` and then down on L208 we can reference `args.minimal`. Currently its getting the args namespace object as the `minimal` argument which is truthy and therefore forcing the minimal output even if the option isn't set.

Reply:
Hope I understood it correctly, check the latest commit.

Reply:
Yep, looks good :+1: